corehq/apps/reports/static/reports/v2/js/columns.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * todo add docstring
+ */
+
+hqDefine('reports/v2/js/columns', [
+  'lodash',  // can we switch to lodash?
+  'reports/v2/js/formatters',
+], function (
+    _,
+    Formatters
+) {
+  'use strict';
+
+  var Column = function (data) {
+    var self = this;
+    self.title = data.title;
```

Comment:
Another newish best practice is to use the assert_properties [like this](https://github.com/dimagi/commcare-hq/commit/46746fabd638d7d73a415e35acded98dda3af331#diff-aed47ded0cafd782cc2aa95c1d1f3341R15) to verify that options objects like this contain the properties they're supposed to.

Reply:
fixed here https://github.com/dimagi/commcare-hq/pull/22063/commits/4769038b326df961c431997f51a4ad2cefff5730